cmd/svcat/binding/get_cmd.go

```diff
@@ -71,7 +76,7 @@ func (c *getCmd) getAll() error {
 		return err
 	}
 
-	output.WriteBindingList(c.Output, bindings.Items...)
+	output.WriteBindingList(c.Output, c.outputFormat, bindings)
```

Comment:
👍 Yeah I get what you were saying before now about needing to switch this to use the root object and not just items.